compiled_starters/csharp/src/Program.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+
+// You can use print statements as follows for debugging, they'll be visible when running tests.
+Console.WriteLine("Logs from your program will appear here!");
+
+// Uncomment this block to pass the first stage
```

Comment:
@ryan-gang can we make the uncommented section shorter? This appears in the UI, so ideally it'd be a small diff, like the Python one:  

<img width="829" alt="Screenshot 2024-04-09 at 16 19 02" src="https://github.com/codecrafters-io/build-your-own-grep/assets/3893573/2bb59b8c-39fc-41c2-9b35-3ee44f476f54">